cypress.config.ts

```diff
@@ -35,6 +35,7 @@ export default defineConfig({
   qaMothPlotUnitStudent5: "/?appMode=qa&fakeClass=5&fakeUser=student:5&qaGroup=5&problem=1.1&unit=./demo/units/qa-moth-plot/content.json",
   qaNoSectionProblemTabUnitStudent5: "/?appMode=qa&fakeClass=5&fakeUser=student:5&qaGroup=5&problem=1.1&unit=./demo/units/qa-no-section-problem-tab/content.json",
   clueTestqaUnitStudent5: "/?appMode=demo&demoName=CLUE-Test&fakeClass=5&fakeUser=student:5&problem=1.1&unit=./demo/units/qa/content.json&noPersistentUI",
+  clueTestNoUnitStudent5: "/?appMode=demo&demoName=CLUE-Test&fakeClass=5&fakeUser=student:5&problem=1.1&noPersistentUI",
```

Comment:
I added this because the existing sets of params did not include a set that will show a student the new sort view. The new tests for document accessibility check whether documents are rendered as accessible in both the large thumbnail view and the smaller "box" view, and it seems the QA unit will only show documents using the large thumbnail view.